x-pack/plugins/security_solution/scripts/endpoint/README.md

```diff
@@ -13,52 +13,10 @@ Example command sequence to get ES and kibana running with sample data after ins
 
 `yarn es snapshot` -> starts ES
 
-`npx yarn start --xpack.securitySolution.enabled=true --no-base-path` -> starts kibana
+`npx yarn start --no-base-path` -> starts kibana. Note: you may need other configurations steps to start the security solution with endpoint support.
```

Comment:
`--xpack.ingestManager.enabled=true` and `--xpack.ingestManager.fleet.enabled=true` aren't needed here?

Reply:
I _think_ `xpack.ingestManager.enabled` might default to `true` and it automatically turns on fleet as well.

Reply:
@michaelolo24 Right now other stuff is needed, but I don't want to host documentation about that here. When that stuff changes, I'd rather document it in one place. I think we used to have a README file, but I don't see it now.

We could add instructions to this file: https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/README.md and then link to that file from this one.

Reply:
If it's defaulted, then no need :) Was just making sure.